PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetCoreEmcal.h

```diff
@@ -98,25 +99,31 @@ class AliAnalysisTaskJetCoreEmcal : public AliAnalysisTaskEmcalJet {
 	Float_t fJetEtaMax;
 	Float_t fJetHadronDeltaPhi;
 	TString fJetContName;
+	Bool_t fRunAnaAzimuthalCorrelation;
 	//
 	TRandom3 *fRandom;
```

Comment:
You should probably specify this and above variables as streaming, but this would be fine in another PR